src/app/Components/Details/QuickAccess/SharedPlan/SharedPlansList/index.jsx

```diff
@@ -0,0 +1,43 @@
+import React, { useEffect, useState } from 'react';
+import findSharedPlans from '../../../../../Gateways/FindSharedPlans';
+import styles from './SharedPlansList.module.scss';
+
+const SharedPlansList = ({ customerId }) => {
+  const [state, setState] = useState({
+    plans: [],
+    loading: true,
+    error: null
+  });
+
+  useEffect(() => {
+    const fetch = async () => {
+      try {
+        const plans = await findSharedPlans({ customerId });
+        setState({ plans, loading: false });
+      } catch (error) {
+        setState({ plans: [], error, loading: false });
+      }
+    };
+
+    fetch();
+  }, [customerId]);
+
+  if (state.plans.length === 0) {
+    return null;
+  }
+
+  return (
+    <div className={styles.wrapper}>
+      <h4 className={styles.heading}>Existing plans</h4>
+      <ul>
+        {state.plans.map(({ id, location }) => (
+          <li key={id}>
+            <a href={location}>{id}</a>
```

Comment:
The IDs that get generated are not super pretty, is there anything else we could display instead?

Reply:
Yeah that is a good point, any suggestions on what we could show instead? Maybe the date it was created - or maybe the goal for the plan? (but I expect that would be too long to fit into this box 😞)

Reply:
I can't think of anything good either :< Maybe just 'Shared plan 1', 'Shared plan 2' ... based on the creation date? (I don't know if we're storing the date btw)

Reply:
Yeah we do store the date, but it doesn't get passed along. I think I like that suggestion for now though, just having "Plan 1", "Plan 2" etc.

I'll quickly change that.